playground/custom-application-config.mjs

```diff
@@ -36,6 +36,7 @@ const config = {
   headers: {
     csp: {
       'connect-src': ['${env:HOST_GCP_STAGING}'],
+      'script-src': ["'unsafe-eval'"],
```

Comment:
I am really uncomfortable with this CSP being set, do you know why `unsafe-eval` is necessary here? [Using eval has large security and performance issues](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_direct_eval!)

Reply:
That line was added originally to help the "Build playground with coverage" step in the workflow pass. But that part of the workflow has changed since then so I was able to remove it now and still have it work as expected.